### PR TITLE
[#159] remove english precedents from raw data

### PR DESCRIPTION
**Pull request by TaimoorRana**
[#159]

- [x] Found and removed 154 English precedents
- [x] manually checked 20 random precedents and they were indeed English

### CODE REVIEW

File: src/ml_service/util/precendent_directory_cleaner.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    """
+    :param directory_path: directory path to search in
+    :param language: language that needs to be removed
+    :return: total files removed
+    """
+    count = 0
+    for filename in os.listdir(directory_path):
+        if filename.endswith(".txt"):
+            file_content = open(directory_path + filename, "r", encoding="ISO-8859-1").read()
+            if is_language_type(file_content, language):
```
Comment (arekmano) [inline src/ml_service/util/precendent_directory_cleaner.py:28]: if detect(file_content) == language

no need for additional method

Comment (TaimoorRana) [reply src/ml_service/util/precendent_directory_cleaner.py:28]: I agree but It's was more to separate the logic of deleting a file from detecting a certain language.

Comment (arekmano) [reply src/ml_service/util/precendent_directory_cleaner.py:28]: IMO no need to have layers of abstraction. I don't foresee us ever having to reuse this code

**Review by dodoels — APPROVED** [review]
LGTM functionally

File: src/ml_service/util/precedent_directory_cleaner_test.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import unittest
+import util.precendent_directory_cleaner as directory_cleaner
+from util.constant import Path
+
+
+class TestLogger(unittest.TestCase):
```
Comment (Samuel-Campbell) [inline src/ml_service/util/precedent_directory_cleaner_test.py]: change class name

Comment (TaimoorRana) [reply src/ml_service/util/precedent_directory_cleaner_test.py]: Good catch! thanks

**Pull request merged**